modules/cnv-creating-storage-class.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-configuring-local-storage-for-vms.adoc
+
+[id="cnv-creating-storage-class_{context}"]
+= Creating a StorageClass object
+
+When you create a StorageClass object, you set parameters that affect the
+dynamic provisioning of PersistentVolumes (PVs) that belong to that storage class. +
+[NOTE]
+====
+You cannot update a StorageClass object's parameters after you create it.
+====
+
+.Procedure
+
+. Create a YAML file for defining the storage class. For example:
++
+----
+$ touch storageclass.yaml
+----
+
+. Edit the file. For example:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: hostpath-provisioner <1>
+provisioner: kubevirt.io/hostpath-provisioner
+reclaimPolicy: Delete <2>
+volumeBindingMode: WaitForFirstConsumer <3>
```

Comment:
we probably need to add `volumeMode: Filesystem` here.
[HCO installs a configmap](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/5e2b026673244c92cbd148badbf00c9c82001578/pkg/controller/hyperconverged/hyperconverged_controller.go#L1045-L1074) causing the default volumeMode to be Block in some cases*.

This is one place where we can override this user interface default.

*"Bare metal platform"

Reply:
If unspecified the default is FileSystem, so no need to specify unless you want something besides the default.

Reply:
Based on @awels comment, I am resolving this without changes.